### DISCUSSION TITLE
Lentidão ao solicitar relatório

### DISCUSSION THREAD

**Discussion by j7info** [RamonSilva20/mapos#1846] [Q&A]
Instalei em ambiente de desenvolvimento Linux. Está funcionando muito bem, até solicitar qualquer relatório. Relatório d produtos chega a demorar mais de 5 min para aparecer. Sabe me dizer o que está causando isso?

**Comment by j7info** [comment]
> Instalei em ambiente de desenvolvimento Linux. Está funcionando muito bem, até solicitar qualquer relatório. Relatório d produtos chega a demorar mais de 5 min para aparecer. Sabe me dizer o que está causando isso?

Pessoal. Instalei em outra VM na minha rede, e a princípio tava funcionando muito bem, gerou os relatório bem rápido, porém coloquei para acessar de fora da minha rede, aí fui gerar e demorou. Se alguém tiver uma luz quanto a isso, agradeço...

**Comment by Pr3d4dor** [comment]
@j7info Conseguiu resolver?

**Comment by j7info** [comment]
> @j7info Conseguiu resolver?

Opa... Não... Ainda não
Descobri que tem algo a ver com as requisições ao banco de dados.
Já aumentei, mas continua.
Quando acesso na rede local, usando o IP interno, funciona 100%, o relatório é gerado em milissegundos... Mas quando acesso usando o IP público, mesmo estando na rede local, demora mais de 3 minutos para gerar o relatório.

**Comment by Pr3d4dor** [comment]
@j7info Estranho, mas só pode ser algo relacionado à sua rede, porque se fosse problema no banco ou mapos iria ser lento no local também.

**Reply by j7info** [reply]
Rapaz, não sei... Mas as outras funcionalidades funcionam 100%
Somente quando solicita um relatório acontece isso...
